assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -253,288 +253,277 @@ db.schema.flyway.scripts.prefix=
-# true. No updates to the namespaces are done if they already conform to the labeling requirements.
+# `true`. No updates to the namespaces are done if they already conform to the labeling requirements.
 che.infra.kubernetes.namespace.label=true
 
-# Defines whether che-server should try to annotate the workspace namespaces.
+# Defines whether {prod-short} server should try to annotate the workspace namespaces.
 che.infra.kubernetes.namespace.annotate=true
 
 # List of labels to find {orch-namespace} that are used for {prod-short} Workspaces.
-# They are used to:
-#  - find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.annotations`.
-#  - actively label {orch-namespace} with any workspace.
-# NOTE: It is strongly recommended not to change the value of this property because the Che operator relies on these labels 
-# and their precise values when reconciling DevWorkspaces. If this configuration is changed, the namespaces will not be automatically
-# recognized by the Che operator as workspace namespaces unless manually labeled as such using the default labels and values.
-# Additional labels on the namespace do not affect the functionality.
+# They perform the following actions:
+#  - Find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.annotations`.
+#  - Actively label {orch-namespace} with any workspace.
+# NOTE: It's strongly recommended not to change the value of this property because the {prod-operator-image-name} relies on these labels 
+# and their precise values when reconciling DevWorkspaces. If this configuration is changed, the namespaces isn't automatically
+# recognized by the {prod-operator-image-name} as workspace namespaces unless manually labeled as such by using the default labels and values.
+# Additional labels on the namespace don't affect the functionality.
 che.infra.kubernetes.namespace.labels=app.kubernetes.io/part-of=che.eclipse.org,app.kubernetes.io/component=workspaces-namespace
 
-# List of annotations to find {orch-namespace} prepared for {prod-short} users workspaces.
-# Only {orch-namespace} matching the `che.infra.kubernetes.namespace.labels` will be matched against these annotations.
+# List of annotations to find {orch-namespace} prepared for {prod-short} user workspaces.
+# Only {orch-namespace} matching the `che.infra.kubernetes.namespace.labels` is matched against these annotations.
 # {orch-namespace} that matches both `che.infra.kubernetes.namespace.labels` and `che.infra.kubernetes.namespace.annotations`
-# will be preferentially used for User's workspaces.
+# is used preferentially for user workspaces.
 # It's possible to use `<username>` placeholder to specify the {orch-namespace} to concrete user.
-# They are used to:
-#  - find prepared {orch-namespace} for users in combination with `che.infra.kubernetes.namespace.labels`.
-#  - actively annotate {orch-namespace} with any workspace.
+# They perform the following actions:
```

Comment:
see above; the annotations don't find and annotate; they're used to find and annotate.